cron/daily-message.js

```diff
@@ -1,6 +1,45 @@
 const moment = require('moment');
 const _ = require('lodash');
+function createMessage({dateString, waitingFor}) {
+  if (!dateString) {
+    return;
+  }
+  const date = moment(dateString).add(5, 'days');
+  const dayDiff = moment().diff(date, 'days');
+  
+  if (dayDiff === -11) {
+    return `Release week starts next week on ${moment(dateString).format('YYYY-MM-DD')} are we all prepared? :lts:`;
+  }
 
+  if (dayDiff === -4) {
+    return `:tada: It's release week!! :tada: To see who is done and who isn't you can say '!release done' and I'll tell you who still needs to do something!
+
+If you know a team is done you can say '!release done <team>' where team can be blog, framework, data, or cli`;
```

Comment:
Let's use `.join()` to break a long string into sentences, but preserve the current formatting.

```suggestion
    return [
      `:tada: It's release week!! :tada: To see who is done and who isn't you can say '!release done' and I'll tell you who still needs to do something!`,
      '',
      `If you know a team is done you can say '!release done <team>' where team can be blog, framework, data, or cli`,
    ].join('\n');
```